net-dialup/picocom/picocom-2024.07.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit bash-completion-r1 toolchain-funcs
+
+MY_PV=${PV/./-}
+
+DESCRIPTION="minimal dumb-terminal emulation program"
+HOMEPAGE="https://gitlab.com/wsakernel/picocom"
+SRC_URI="https://gitlab.com/wsakernel/${PN}/-/archive/${MY_PV}/${PN}-${MY_PV}.tar.bz2"
+
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
```

Comment:
Since this is so old and most c files have been touched, let's better just drop all keywords except `~amd64` and file a rekeywording bug for the rest.  We can also keyword `dev-go/go-md2man` at the same time and avoid adding stuff to package.use.mask.